Tools/scripts/pathfix.py

```diff
@@ -164,12 +180,28 @@ def fix(filename):
     # Return success
     return 0
 
+
+def parse_shebang(shebangline):
+    end = len(shebangline)
+    start = shebangline.find(b' -', 0, end)  # .find() returns index at space
+    if start == -1:
+        return b'', b''
+    print(shebangline[(start + 2):end])
+    return shebangline[(start + 2):end]
+
+
 def fixline(line):
+    global keep_flags
```

Comment:
I don't think that this global statement is useful. Variables are global by default.